google-cloud-spanner/clirr-ignored-differences.xml

```diff
@@ -1,662 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Don't we need this anymore?

Reply:
Once the version with the breaking changes is published, the clirr check won't detect the same changes as breaking again. So the clirr-ignored-differences file is no longer required to ignore the same breaking changes.

Reply:
I see, but it is nice to have this file with some examples if we have to add any exceptions. Could we leave a few entries here? Or is this automatically deleted?